generator/api/template/src/test/test.js

```diff
@@ -3,40 +3,16 @@ import 'babel-polyfill';
 import assert from 'assert';
 import GitHubUser from '../lib/GitHubUser';
 
-// WARNING: don't use arrow functions with mocha describe(), it(),
```

Comment:
Do you feel strongly about removing my arrow function warning? I wanted to leave that in for people to explain why I use function. The actually function declaration was embedded in my async wrapper. Since we're removing the async wrapper now, the `it` calls should use `function` and not arrow syntax to avoid users from being surprised that they aren't in the mocha context on callback.